runtimes/eden/src/migrations.rs

```diff
@@ -0,0 +1,77 @@
+use crate::Runtime;
+use frame_support::{migration, parameter_types, traits::OnRuntimeUpgrade, weights::Weight, BoundedVec};
+use primitives::AccountId;
+
+#[cfg(feature = "try-runtime")]
+use frame_support::traits::OnRuntimeUpgradeHelpersExt;
+
+// MaxMembers is chosen based on what used to be the MaxMembers param for the pallet ValidaorsSet
+// We have intentionally used the same number for MaxInvulnerables for the pallet CollatorSelection
+// So the BoundedVec storage in these two pallets remain compatible.
+parameter_types! {
+	pub const MaxMembers: u32 = 50;
+}
+
+pub struct MoveValidatorsSetToInvulnerables;
+impl OnRuntimeUpgrade for MoveValidatorsSetToInvulnerables {
+	fn on_runtime_upgrade() -> Weight {
+		if let Some(validators) =
+			migration::take_storage_value::<BoundedVec<AccountId, MaxMembers>>(b"ValidatorsSet", b"Members", &[])
+		{
+			log::info!(target: "try-runtime", "ValidatorsSet::Members are {:?}", validators);
```

Comment:
```suggestion
			log::info!(target: "migration", "ValidatorsSet::Members are {:?}", validators);
```
Technically this isn't run only in the `try-runtime` target but also on a real migration